core/soc/estimator.go

```diff
@@ -107,10 +119,6 @@ func (s *Estimator) SoC(chargedEnergy float64) (float64, error) {
 	if fetchedSoC == nil {
 		f, err := s.vehicle.SoC()
 		if err != nil {
-			if errors.Is(err, api.ErrMustRetry) {
```

Comment:
damit wir errmustretry jetzt immer verschlucket- für apis mit refreshkey ein nogo.

Reply:
Also bei Chargern eine andere Logik als bei Vehicle API? Warum? Wie macht man das im Code klar warum?

Reply:
Habs beim Charger mal entfernt, in der Vermutung dass wir das nicht brauchen werden. (Keine Ahnung ob das nicht mal anders sein kann)

Und für die Vehicle API einen Kommentar hinzugefügt, dass man auch weiss warum das dort so ist. Denn von der Logik ist das nicht klar.